src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -35,7 +35,7 @@ public class NodeFunctionInvoker : FunctionInvokerBase
         private readonly string _entryPoint;
 
         private Func<object, Task<object>> _scriptFunc;
-        private Func<object, Task<object>> _clearRequireCache;
+        private static Func<object, Task<object>> _clearRequireCache;
```

Comment:
This is not required, but an enhancement to make sure we don't have this done/loaded for every `NodeFunctionInvoker` instance.